docs/src/contributing-to-popochiu/contributing-documentation/basic-rules.md

```diff
@@ -0,0 +1,87 @@
+
+- **Tutorials**: Step-by-step guides that show how stuff work (**learning**)
+- **How-to Guides**: Focused, task-oriented instructions to achieve a certain goal . think recipes (**achieving a goal**).  
+- **Explanations**: Conceptual overviews and background information (main goal: **understanding**)
+- **Reference**: Well-organized list of techical details and descriptions (main goal: **finding information**)
+
+### Documentation Sections and Content Types
+
+Each section in Popochiu's documentation aligns with a type from the Diátaxis framework. Here's a quick overview:
+
+| **Section**              | **Content types**        | **Purpose**                                                                                                               |
+|--------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| Getting Started          | Tutorials, Explanations  | Provides the basics for installing Popochiu, taking first steps, and joining the community.                               |
+| The Editor Handbook      | References, Tutorials    | Showcases the functionality of the Popochiu plugin, explains how it works, and serves as a go-to guide for its use.       |
+| The Engine Handbook      | References, Explanations | Explains the plugin's architecture and provides a comprehensive API reference for game developers.                        |
+| How to Develop a Game    | Tutorials, How-To        | Guides users through setting up their first game, learning by doing, and building confidence to explore advanced features. |
+| Advanced Techniques      | Tutorials, How-To        | Offers in-depth guides and practical examples for getting the most out of Popochiu (and beyond).                          |
+| Contributing to Popochiu | Explanations, References | Provides all the information contributors need to get involved in Popochiu development.                                    |
+
+If you add new stuff, make sure you put in in the right place. If in doubt, reach out for **@stickgrinder**.
+
+---
+
+## Contributing to missing documentation
+
+Some areas of the documentation are marked as **TODO**. If you think you know how to complete them (yay!):
+
+1. Draft an outline for the missing content.
+2. Commit the outline and open a draft PR.
+3. Assign **@stickgrinder** as a reviewer.
+4. Incorporate feedback and proceed with the full write-up.
+
+If you can't - or don't want to incorporate all the feedback (for example if we ask you to expand the docs too much and you are unable to provide all the necessary work), we can still cooperate to complete it together.
```

Comment:
I love this!!! ❤️❤️❤️

Reply:
Thanks! I mentioned myself so I can own this and act as a proxy for you and you don't get too distracted from other stuff. I'll reach out for you anytime for validation, if that's something non-trivial, of couse.